packages/amplify-migration-template-gen/src/template-generator.ts

```diff
@@ -57,10 +58,23 @@ class TemplateGenerator {
     const gen2CategoryStacks = destStackResources?.filter((stackResource) => stackResource.ResourceType === CFN_RESOURCE_STACK_TYPE);
     assert(gen1CategoryStacks && gen1CategoryStacks?.length > 0, 'No gen1 category stack found');
     assert(gen2CategoryStacks && gen2CategoryStacks?.length > 0, 'No gen2 category stack found');
-    gen1CategoryStacks.forEach(({ LogicalResourceId: Gen1LogicalResourceId, PhysicalResourceId: Gen1PhysicalResourceId }) => {
+    for (const { LogicalResourceId: Gen1LogicalResourceId, PhysicalResourceId: Gen1PhysicalResourceId } of gen1CategoryStacks) {
       const category = CATEGORIES.find((category) => Gen1LogicalResourceId?.startsWith(category));
-      if (!category) return;
+      if (!category) continue;
       assert(Gen1PhysicalResourceId);
+      if (category === 'auth') {
+        const { StackResources: AuthStackResources } = await this.cfnClient.send(
+          new DescribeStackResourcesCommand({
+            StackName: Gen1PhysicalResourceId,
+          }),
+        );
+        assert(AuthStackResources);
+        if (AuthStackResources.some((authResource) => authResource.ResourceType === CFN_AUTH_TYPE.UserPoolGroups)) {
```

Comment:
this is a temporary workaround to unblock testing.  We will be supporting user pool groups as part of refactor.